lib/dal/src/audit_logging.rs

```diff
@@ -217,7 +216,7 @@ async fn assemble_for_list(
     audit_log: AuditLog,
 ) -> Result<Option<FrontendAuditLog>> {
     match audit_log {
-        AuditLog::V3(inner) => {
+        AuditLog::V1(inner) => {
```

Comment:
V3 to V1? Just surprising

Reply:
Yes. Ideally, we would have went from `VBeta3` to `V1` or something analogous. We are wholesale dropping the old beta streams in favor of the production ones, so this is a good time to reset the counter.